front/components/V2/TabNavigationButton.tsx

```diff
@@ -0,0 +1,86 @@
+import { View } from 'react-native';
+import { Pressable, Text } from 'native-base';
+import React from 'react';
+
+type TabNavigationButtonProps = {
+	icon?: React.ReactNode;
+	label: string;
+	onPress: () => void;
+	onLongPress: () => void;
+	isActive: boolean;
+	isCollapsed: boolean;
+};
+
+const TabNavigationButton = (props: TabNavigationButtonProps) => {
+	return (
+		<Pressable
+			onPress={props.onPress}
+			onLongPress={props.onLongPress}
+			style={{
+				width: '100%',
+			}}
+		>
+			{({ isPressed, isHovered }) => (
+				<View
+					style={{
+						display: 'flex',
+						flexDirection: 'row',
+						alignSelf: 'stretch',
+						alignItems: 'center',
+						justifyContent: 'flex-start',
+						padding: '10px',
+						borderRadius: 8,
+						flexGrow: 0,
+						// @ts-expect-error BoxShadow is not in the types but I want it this may be a legitimate error on my part
+						boxShadow: (() => {
+							if (isHovered) {
+								return '0px 0px 16px 0px rgba(0, 0, 0, 0.25)';
+							} else if (props.isActive) {
+								return '0px 0px 8px 0px rgba(0, 0, 0, 0.25)';
+							} else {
+								return undefined;
+							}
+						})(),
+						backdropFilter: 'blur(2px)',
+						backgroundColor: (() => {
```

Comment:
Couldn't these values be used elsewhere? Shouldn't they be in the global theme? Especially if they are raw CSS

Reply:
Actuellement non Mathys à un component pour faire ça cool mais par manque de temps j'applique directement les styles de boxshadow en fonction d'un scénario 

Dans l'idée oui, dans cette pr non